src/Template/Element/advanced_search_form.ctp

```diff
@@ -30,6 +30,21 @@ echo $this->Form->create('AdvancedSearch', [
 ?>
 
 <div layout="column" ng-app="app" ng-cloak>
+    <?php if ($this->request->getParam("?") && !(isset($isSidebar) && $isSidebar)): ?>
+        <div id="template-notif">
+            <i><?= __("This form has been filled by a template.")?></i>
```

Comment:
I wonder if this sentence is very clear. The way I see it, the pre-filled form *is* the template. I suggest *This form is prefilled.*